en_US/configuration/logs.md

```diff
@@ -73,28 +73,12 @@ Where,
 
```

Comment:
@wivwiv , please help to check if `file_handlers.default` needs to be listed here. It is not in the configuration example above.

Reply:
No need, this is the current configuration for logging, which has been simplified by removing file_handler.default
```md
log {
  audit {
    enable = true
    ignore_high_frequency_request = true
    max_filter_size = 5000
    path = "/Users/emqx/Downloads/emqx-560/log/audit.log"
    rotation_count = 10
    rotation_size = 50MB
    time_offset = system
    timestamp_format = auto
  }
  console {
    enable = true
    formatter = json
    level = debug
    time_offset = system
    timestamp_format = auto
  }
  file {
    default {
      enable = true
      formatter = text
      level = warning
      path = "/Users/emqx/Downloads/emqx-560/log/emqx.log"
      rotation_count = 10
      rotation_size = 50MB
      time_offset = system
      timestamp_format = auto
    }
  }
  throttling {time_window = 1m}
}
```